From 32adca93aa527cd019655436ab98d6207ef9573e Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Wed, 21 Jun 2023 19:15:57 +0200
Subject: [PATCH] Port to OVOS

Mycroft has seized development due to running out of money. the
OpenVoiceOS (OVOS) community has taken over development and so this
commit ports the application to their stack. It's a drop-in replacement,
just needs changing of imports
---
 src/daemon.py          | 9 +++++----
 src/types/skill.py     | 2 +-
 src/views/pair.py      | 6 +++---
 src/wrappers/dialog.py | 2 +-
 4 files changed, 10 insertions(+), 9 deletions(-)

diff --git a/src/daemon.py b/src/daemon.py
index 1cd8ca2..4d6bd01 100644
--- a/src/daemon.py
+++ b/src/daemon.py
@@ -2,10 +2,11 @@
 """
 Contains code for the message bus daemon.
 """
-from mycroft.api import DeviceApi, check_remote_pairing
-from mycroft.util.signal import create_signal
-from mycroft_bus_client import Message
-from mycroft_bus_client import MessageBusClient as OriginalMessageBusClient
+from ovos_backend_client.api import DeviceApi
+from ovos_backend_client import check_remote_pairing
+from ovos_utils.signal import create_signal
+from ovos_bus_client import Message
+from ovos_bus_client import MessageBusClient as OriginalMessageBusClient
 from gi.repository import Gio, GObject
 import socket
 import time
diff --git a/src/types/skill.py b/src/types/skill.py
index c181f22..03f3f8d 100644
--- a/src/types/skill.py
+++ b/src/types/skill.py
@@ -2,7 +2,7 @@
 """
 Contains code for handling skills.
 """
-from mycroft.configuration import Configuration
+from ovos_config import Configuration
 from gi.repository import Gio, GLib, Gtk, GObject
 import pathlib
 import os.path
diff --git a/src/views/pair.py b/src/views/pair.py
index c50c8b9..325e693 100644
--- a/src/views/pair.py
+++ b/src/views/pair.py
@@ -6,9 +6,9 @@ from gi.repository import Adw, Gtk
 from uuid import uuid4
 from requests import HTTPError
 import threading, time
-from mycroft_bus_client import Message
-from mycroft.api import is_paired
-from mycroft.identity import IdentityManager
+from ovos_bus_client import Message
+from ovos_backend_client import is_paired
+from ovos_backend_client.identity import IdentityManager
 import os
 
 from ..daemon import get_daemon
diff --git a/src/wrappers/dialog.py b/src/wrappers/dialog.py
index 202ad1c..0acfad2 100644
--- a/src/wrappers/dialog.py
+++ b/src/wrappers/dialog.py
@@ -2,7 +2,7 @@
 """
 Common wrappers.
 """
-from mycroft_bus_client import Message
+from ovos_bus_client import Message
 from gi.repository import Gtk
 
 class DialogWrapper(Gtk.Box):
-- 
2.41.0

